registration/src/bot_class.py

```diff
@@ -350,6 +352,29 @@ async def permissions_(
         async def whois_printing_(
                 interaction: discord.Interaction):
             return await who_is_printing(interaction)
+        
+        @self.tree.command(
+            name="add-mac-address",
+            description="**ADMIN ONLY**: Add a BLE mac address to trained committee"
+        )
+        @committee_command
+        async def add_mac_address_(
+            interaction : discord.Interaction,
+            user : str,
+            mac_addr : str
+        ):
+            return await add_mac_address(interaction, user, mac_addr)
+        @self.tree.command(
+            name="trained-member-present",
+            description="**ADMIN ONLY**: Check if a lab trained member is present in a time interval"
+        )
+        @committee_command
+        async def trained_member_present_(
+            interaction: discord.Interaction,
+            timestamp: str,
+            interval: int,
+        ):
+            return await trained_member_present(interaction, timestamp, interval)
```

Comment:
is this needed? might be nicer to have a status page + webhook like how the printer streaming stuff is done?